scss/_skeletons.scss

```diff
@@ -0,0 +1,60 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
```

Comment:
This is already well explained in the docs, no need to mention this in the sources :)